khelo/lib/ui/flow/stats/my_stats_tab_screen.dart

```diff
@@ -107,6 +95,17 @@ class _MyStatsTabScreenState extends ConsumerState<MyStatsTabScreen>
             selected: _selectedTab == 1,
             onTap: () => _controller.jumpToPage(1),
           ),
+
+          // Dummy add-icon to maintain consistency in tab placement for myCricket & Stats tab.
+          Visibility(
+            visible: false,
+            maintainState: true,
+            maintainAnimation: true,
+            maintainSize: true,
+            child: actionButton(context,
+                onPressed: null,
+                icon: Icon(Icons.add, color: context.colorScheme.primary)),
+          ),
```

Comment:
I think this icon unused.

Reply:
yes, it is to resolve that tab-placement issue, added comment to clarify it's purpose.